index.js

```diff
@@ -73,6 +73,8 @@ Cypress.Commands.add('percySnapshot', (name, options) => {
         // Handle errors
         log.error(`Could not take DOM snapshot "${name}"`);
         log.error(error);
+        /* istanbul ignore next */
```

Comment:
- I tried to write test to cover throwing of error, but was unable to do this.
- It is little complicated in cypress test to test this.

Reply:
issue -> https://github.com/cypress-io/cypress/issues/2487. there is no definite answer here. Maybe we can sync on this.